spec/features/workflow/schedule_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+RSpec.feature "Scheduling an edition" do
+  scenario do
+    given_there_is_an_edition_submitted_for_review
+    when_i_visit_the_summary_page
+    and_i_click_on_schedule_publishing
+    then_i_see_a_default_date_and_time_selected
+
+    when_i_fill_in_a_valid_date
+    and_i_select_a_valid_time
+    and_i_click_save
+    then_i_see_the_edition_has_been_scheduled
```

Comment:
I think we should use a term other than 'scheduled', as (at least right now) the user has only set the date/time by doing this.

Reply:
Now changed to `then_i_see_the_edition_has_a_set_scheduled_publishing_date_and_time`